<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>show_composerEtparticiper</class>
 <widget class="QDialog" name="show_composerEtparticiper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>503</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Gabriola</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Back up your data with us :</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Séléctionner une projet : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_projet"/>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Séléctionner une tâche  :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_tache"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView_addtaskt_into_project"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="pushButton_add_task_into_project_From_Dialog">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777214</width>
             <height>30</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 170, 0);
border-color: rgb(0, 170, 0);
border-width:1px;
border-style:solid;
border-radius:3px;</string>
           </property>
           <property name="text">
            <string>Ajouter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_supp_composer">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">
background-color: rgb(255, 48, 7);
border-color: rgb(255, 48, 7);
border-width:1px;
border-style:solid;
border-radius:3px;</string>
           </property>
           <property name="text">
            <string>Supprimer</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Gabriola</family>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Back up your data with us !</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Séléctionner une tâche :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_tache_pour_des_Intervs"/>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Séléctionner un intervenant :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_Interv_qui_participe"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tableView_add_interv_into_Task"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_ajouter_un_interv">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 170, 0);
border-color: rgb(0, 170, 0);
border-width:1px;
border-style:solid;
border-radius:3px;</string>
           </property>
           <property name="text">
            <string>Ajouter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_supp_participer">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">
background-color: rgb(255, 48, 7);
border-color: rgb(255, 48, 7);
border-width:1px;
border-style:solid;
border-radius:3px;
</string>
           </property>
           <property name="text">
            <string>Supprimer</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
